source/appModules/powerpnt.py

```diff
@@ -587,38 +587,70 @@ def _getOverlapText(self):
 			if overlapsOtherLeftBy>0:
 				total=False
 				if otherIsBehind:
-					# Translators: A message when a shape is infront of another shape on a Powerpoint slide 
-					textList.append(_("covers left of {otherShape} by {distance:.3g} points").format(otherShape=otherLabel,distance=overlapsOtherLeftBy))
+					textList.append(ngettext(
+						# Translators: A message when a shape is infront of another shape on a Powerpoint slide
```

Comment:
Correct spelling of 'in front', and proper capitalization for PowerPoint:
```suggestion
						# Translators: A message when a shape is in front of another shape on a PowerPoint slide
```